game_patch/rf/player/camera.h

```diff
@@ -45,4 +45,6 @@ namespace rf
         AddrCaller{0x0040D780}.c_call(&result, camera);
         return result;
     }
+
+    static auto& camera_get_mode = addr_as_ref<CameraMode(rf::Camera*)>(0x0040D740);
```

Comment:
```suggestion
    static auto& camera_get_mode = addr_as_ref<CameraMode(const rf::Camera&)>(0x0040D740);
```

Reply:
This causes a build error.

Reply:
Call it with a reference. 
```C++
rf::camera_get_mode(*rf::local_player->cam);
```

This function has an invariant that its parameter is not null.